src/controllers/UsersController.ts

```diff
@@ -33,7 +33,7 @@ export const getUser = async (req: Request, res: Response): Promise<Response> =>
 export const createUser = async (req: Request, res: Response): Promise<Response> => {
     var {name, email, password, phone_num, date_of_birth, race, gender, address, employemnt_status,
         military_affiliated, military_affiliation, military_start_date, military_end_date, last_rank,
-        milirary_speciality, household_size, income, current_course, completed_courses} = req.body;
+        milirary_speciality, household_size, income, current_course, completed_courses, refferal_source, resources} = req.body;
```

Comment:
Can we set (date_of_birth, race, gender, address, military_start_date, military_end_date, last_rank, milirary_speciality, household_size, income, current_course, completed_courses) to a default value instead of forcing the front-end client to specify this?

The register page only have the following input fields: name, email password, military affiliation, employment status so we won't be able to come up with a value for the other fields.